Gemfile.lock

```diff
@@ -66,6 +70,7 @@ GEM
 
 PLATFORMS
   ruby
+  x64-mingw32
```

Comment:
Not a huge deal but I assume you were only able to get this to build locally with mingw installed?